cfgov/unprocessed/css/organisms/content-main.less

```diff
@@ -0,0 +1,24 @@
+/* topdoc
+  name: Content Main
+  family: cfgov-organism
+  patterns:
+    - name: Default example
+      markup: |
+          <div class="o-content-main">
+              <*></*>
+          </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-content-main
+  tags:
+    - cfgov-organism
+*/
+
+.o-content-main {}
```

Comment:
yeah, it's just a placeholder. I'm going to use it in https://github.com/cfpb/cfgov-refresh/pull/2785.